lib/server/graphql/post-workers/BaseGraphqlPostWorker.js

```diff
@@ -46,6 +48,63 @@ export default class BaseGraphqlPostWorker {
       engine,
     })
   }
+
+  /**
+   * get: Name of the operation.
+   *
+   * @abstract
+   * @returns {string} - Operation name.
+   * @throws {ConcreteMemberNotFoundGraphqlError} - If this method is not implemented.
+   */
+  static get schema () {
+    throw ConcreteMemberNotFoundGraphqlError.create({
+      value: {
+        memberName: 'BaseGraphqlPostWorker.get:schema',
+      },
+    })
+  }
+
+  /**
+   * get: Constructor.
+   *
+   * @abstract
```

Comment:
Does this need to be abstracted?

```suggestion
   * @abstract
```

Reply:
@hecticme 

Thank you for your review :)

Fix it
https://github.com/openreachtech/renchan-core/pull/280/commits/5e1cdc59b1e0d278354410b4f5b5efb0b9aaaf9e